app/src/androidTest/java/xyz/jhughes/laundry/Matchers/RecyclerViewAdapterNotNullAssertion.java

```diff
@@ -0,0 +1,28 @@
+package xyz.jhughes.laundry.Matchers;
+
+import android.support.test.espresso.NoMatchingViewException;
+import android.support.test.espresso.ViewAssertion;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import static android.support.test.espresso.matcher.ViewMatchers.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.nullValue;
+
+public class RecyclerViewAdapterNotNullAssertion implements ViewAssertion {
```

Comment:
To clarify, we don't have to do this right away. But I think it would be really cool to have a sigapp testing library.